src/AgentResultMapping.sol

```diff
@@ -9,7 +9,7 @@ contract AgentResultMapping {
         address indexed publisherAddress,
         string[] outcomes,
         uint16[] estimatedProbabilitiesBps,
-        bytes32[] txHashes,
+        bytes[] txHashes,
```

Comment:
After this change, how can we differentiate between Cow_uuid and txHash in Dune?

Reply:
My understanding of https://github.com/gnosis/prediction-market-agent-tooling/pull/771 is that a txHash (i.e. bytes32) will always get passed to this contract, hence no changes required in this PR - or is that not the case?

Reply:
> My understanding of https://github.com/gnosis/prediction-market-agent-tooling/pull/771 is that a txHash (i.e. bytes32) will always get passed to this contract, hence no changes required in this PR - or is that not the case?

Oh wait, that's actually true!

I was working with UID, then found out txHash and then forgot that the hash should be bytes32 🤦 

But will test it, but probably consider only PMAT PR relevant then.

Reply:
> But it's a valid question. I will double check with Dune guys if this is fine with them.

Hugo says they will use tx hash only to get the timestamp of the bet, so they don't care about the kind of transaction it is for.

Reply:
Sure, but if we have the `txHash` always available, why not simply use that?
Also, I believe they can fetch important info from `txHash`, such as `outcome_tokens_purchased`, etc. So I would really advise to use only `txHash` instead of `cow_uid`.